### ISSUE TITLE
A few accessibility problems with gogs

### ISSUE THREAD

**Issue by lilmike** [gogits/gogs#2491]
Hi,
I am blind, so use a screen reader called jaws for windows, though there are others such as NVDA (nonvisual desktop access), and Gogs seems to have a few accessibility issues with the dashboard. The main ones I've noticed are that the menu that opens up to sign out, view admin panel, etc, is not labelled properly. It seems to be an unknown unicode character (unknown to jaws, that is), so it reads it simply as a question mark. It does identify it as clickable, so I know it can be pressed enter on to open a menu or similar, but I do not know what it does without trying it. I am using the latest gogs version as of this writing, arch linux fully updated, and jaws 15.
Thanks for any feedback/fixes.
-Michael.

**Comment by Unknwon** [comment]
Thanks your feedback!

I'll take a look ASAP.

**Comment by Unknwon** [comment]
Hi, I've added some fixes for screen reader on dashboard, though I'm using Mac and can't test it very well.

You can test on https://try.gogs.io/ and give feedback.

**Comment by Unknwon** [comment]
ping @lilmike

**Comment by Unknwon** [comment]
Close due to lack of feedback.
